cognitedata/reveal
# refactor(react-components): improve rule based styling applier

#### Type of change
![Refactor](https://img.shields.io/badge/Type-Refactor-lightgrey) 

## Description :pencil:
Some small improvements on applying the node styles. Before it was applying separately. Now it is applying altogether at the end of the traverse expression cycle. 

## Test instructions :information_source:
- Select a Story with cad model that has contextualized nodes and that could enable rule based coloring rules. 
(tested more often with modelId : 4319392643513894, revisionId : 91463736617758)
- Select a RuleSet.
- Observe the rules are applying. No visual changes but now it is a bit more performant


## Checklist :ballot_box_with_check:
